lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -231,7 +231,7 @@ sub run_cromwell_gcp {
     # Cloudize workflow
     #
     my $cloud_yaml = $yaml; $cloud_yaml =~ s/.ya?ml/_cloud.json/;
-    my $lsb_sub_guard = Genome::Config::set_env('lsb_sub_additional', 'docker(jackmaruska/cloudize-workflow:1.3.1)');
+    my $lsb_sub_guard = Genome::Config::set_env('lsb_sub_additional', 'docker(mgibio/cloudize-workflow:1.1.0)');
```

Comment:
Does the new image's 1.1.0 correspond to the old 1.3.1 or is this a different version?

Reply:
Newer but just adds scripts unrelated to running in GMS. Versioning was very wild-west and I only recently started doing real releases so there isn't really a way to get the old one without detective work. I can push through a test run to confirm behavior if you'd like